homeassistant/components/switch/wemo.py

```diff
@@ -109,12 +110,33 @@ def device_state_attributes(self):
 
         if self.insight_params or (self.coffeemaker_mode is not None):
             attr[ATTR_CURRENT_STATE_DETAIL] = self.detail_state
+            attr['current_power_w'] = \
+                self.insight_params['currentpower'] / 1000
+            attr['today_power_mW_min'] = self.insight_params['todaymw']
+            attr['total_power_mW_min'] = self.insight_params['totalmw']
```

Comment:
This line is duplicated - but it's also a duplicate of the existing `current_power_mwh` property of the switch - so you can just remove these - and depend on the switch attributes.